internal/bminventory/inventory_v2_handlers.go

```diff
@@ -273,3 +275,23 @@ func (b *bareMetalInventory) V2UpdateClusterLogsProgress(ctx context.Context, pa
 
 	return installer.NewV2UpdateClusterLogsProgressNoContent()
 }
+
+func (b *bareMetalInventory) V2DownloadClusterLogs(ctx context.Context, params installer.V2DownloadClusterLogsParams) middleware.Responder {
```

Comment:
lets point DownloadClusterLogs function (V1) to simply call V2DownloadClusterLogs. That way will will also get the unit tests for free, and the code will actually start using V2 API. you can see how i did it in the InstallCluster, for example